src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -610,7 +610,7 @@ public override void GenerateViewGeometryFromRenderPackagesAndRequestUpdate(IEnu
             OnRequestViewRefresh();
         }
 
-        protected override void DeleteGeometryForIdentifier(string identifier, bool requestUpdate = true)
+        internal override void DeleteGeometryForIdentifier(string identifier, bool requestUpdate = true)
```

Comment:
I needed to be able to call this externally so marked it `internal` and calling it from `Watch3DView`. Hope it's okay to break the accessibility here through the new `IWatch3DView` interface. See PR: https://git.autodesk.com/Dynamo/DynamoPro/pull/178